src/v3/src/components/Button/Button.tsx

```diff
@@ -79,24 +81,24 @@ const Button: UISchemaElementComponent<{
 
   return (
     <OdyButton
+      label={label ?? ''}
       type={type}
       variant={variant ?? 'primary'}
-      fullWidth={wide ?? true}
-      ref={focusRef}
-      disabled={loading || disabled}
-      className={classes}
-      // Fixes text overflow
-      sx={{ display: 'flex', whiteSpace: 'normal' }}
-      startIcon={loading ? <Spinner color="white" /> : ButtonImageIcon}
-      aria-describedby={ariaDescribedBy}
-      data-type={dataType}
-      data-se={dataSe}
-      aria-label={ariaLabel}
+      isFullWidth={wide ?? true}
+      buttonFocusRef={focusRef}
+      isDisabled={loading || disabled}
+      startIcon={loading ? (
+        <Spinner
+          color={!variant || variant === 'primary' ? tokens.HueNeutralWhite : undefined}
+        />
+      ) : ButtonImageIcon}
+      ariaDescribedBy={ariaDescribedBy}
+      testId={dataSe ?? dataType}
```

Comment:
Ideally Odyssey would give us some sort of `data-type` attribute but I guess there aren't any cases where we render buttons that need both `data-se` and `data-type`. We should definitely document this in the testing selector changes doc